non-msrv-tests/tests/ui/item.stderr

```diff
@@ -14,38 +14,68 @@ error: empty `derive_where` found
   |
   = note: this error originates in the attribute macro `derive_where` (in Nightly builds, run with -Z macro-backtrace for more info)
 
+error: the `crate` option has to be defined in it's own `#[derive_where(..)` attribute
+  --> tests/ui/item.rs:11:16
+   |
+11 | #[derive_where(crate(derive_where_))]
+   |                ^^^^^
+
+error: expected path, expected identifier
+  --> tests/ui/item.rs:14:24
+   |
+14 | #[derive_where(crate = "struct Test")]
+   |                        ^^^^^^^^^^^^^
+
+error: the `crate` option has to be defined in it's own `#[derive_where(..)` attribute
+  --> tests/ui/item.rs:17:16
+   |
+17 | #[derive_where(crate = "derive_where_", crate = "derive_where_")]
+   |                ^^^^^
+
```

Comment:
I was thinking if this should be a seperate error `duplicate_crate`, but it's a pretty stupid misstake to make, and I'd guess you'd see the mistake if it is marked as a mistake. Just maybe the span should be the whole content and not only the first `crate`. 

Not sure tho.

Reply:
As above, awful error handling code branches.

Reply:
Yeah, I tried, but I couldn't really figure out a clean way without changing a lot of stuff around. Unless you really mind I plan to leave it like that.

Reply:
No I think we can keep it as is.

Reply:
Maybe we can document it in the test case, as not ideal but the best we can easily do here.

Reply:
That sounds like a good idea.

Reply:
Done.